src/Sarif.Multitool/Extensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    public static class Extensions
+    {
+        public static bool RefersToDriver(this ToolComponentReference toolComponent, string driverGuid)
+        {
+            if (toolComponent.Index == -1)
+            {
+                if (string.IsNullOrWhiteSpace(toolComponent.Guid))
```

Comment:
>IsNullOrWhiteSpace [](start = 27, length = 18)

Just `null`. We are checking if `toolComponent.guid` is _absent_ from the SARIF file. If it's absent from the SARIF file, `toolComponent.guid` will be `null`.

Reply:
ok. i will remove the test cases where we test for "", because that won't exist

---
In reply to: [455816703](https://github.com/microsoft/sarif-sdk/pull/1995#discussion_r455816703) [](ancestors = 455816703)